Workbooks/SenservaProMultipleWorkspaceWorkbook.json

```diff
@@ -0,0 +1,132 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "## Using SenservaPro Across Multiple Log Analytics Workspaces\n---\n\nThis workbook helps serve as a guide for users who are using SenservaPro across multiple instances of SenservaPro. These users generally have multiple tenants or a geographical data location requirement. \nMore information can be found at https://docs.microsoft.com/en-us/azure/sentinel/extend-sentinel-across-workspaces-tenants\n\n\nSelect one workspace as the main conduit to create a unified view. Create a Sentinel Function to provide an easy alias for your queries.\nMore information about Sentinel Functions can be found at https://docs.microsoft.com/en-us/azure/azure-monitor/logs/functions"
```

Comment:
In the link country code (en-us) should not mention due to this checks are failing. Please make the required change.

Thanks,
Aditi Maheshwari

Reply:
@v-admahe - This change has been made